### DISCUSSION TITLE
Web Site for the Project

### DISCUSSION THREAD

**Discussion by ZigRazor** [ZigRazor/CXXGraph#200] [Ideas]
Here we can talk about the development of the site for the CXXGraph project

**Comment by luigimorel** [comment]
Hey,
How about using  [docusaurus](https://docusaurus.io/) for the documentation. 

Cheers.

**Comment by ZigRazor** [comment]
Hi @morelmiles,
For the website development you can use the technoligies and frameworks that you prefer. 
Docusaurus can be a good framework.
The important things is that is full-integrated with github-pages ( the site relies on Github-pages).
You can start develop on the branch [gh-pages](https://github.com/ZigRazor/CXXGraph/tree/gh-pages).
I remove the github-actions that influence the update of this branch and in the meantime I add you as collaborator for this project. OK?

Thank you

**Reply by luigimorel** [reply]
Great. 
Yes, Docusaurus integrates easily with gh-pages since it is reactjs based. 

I'll focus on development in two days' time. Let me know what you think. 

Thanks...

**Reply by ZigRazor** [reply]
Perfect, open a pull request, when you have also a draft, so i can see it.
Then we can refine it with time, with a view to doing CI/CD also for the site.

Thanks

**Comment by ZigRazor** [comment]
@morelmiles Starting from your emails.
The base site need this kind of tree(section):
- Home
- How to Install ( 3 subsection for different kind of linux version )
- Algorithms ( with a sub-page for each algorithm )
- Partition Algorithms ( with a sub section for each partition algorithm )
- Guide ( where we will implement some basic examples for the moment and we include the doxygen documentation)
- How to Contribute 
- About us

If you have some questions you can ask me.

Thank you so much,
Cheers

**Comment by luigimorel** [comment]
Great, 
Thanks for the clarification. 

Cheers,

**Comment by luigimorel** [comment]
Hey @ZigRazor, 
I have started working on the website. 
https://github.com/morelmiles/CXXGraph/tree/gh-pages

Do I have to wait until I finish the first draft before i can send in the first draft?
Or we can do it incrementally (merge the changes as they come in)?
Delayed to start because of commitments. 

Cheers,

**Reply by ZigRazor** [reply]
I think we can merge changes, one by one, so in the meantime i can see the result, and get improvements with you😎

**Comment by luigimorel** [comment]
Hey @ZigRazor , 
Can I get a logo that I could use? See attached image.
![image](https://user-images.githubusercontent.com/48476293/161436374-8b0b57bd-1566-4b05-b9e7-45e7746ed6e9.png)

 Cheers

**Reply by ZigRazor** [reply]
Obvously you can get logos by the repos, if you need somenthing different tell me your needs.

Thank you so much!
P.S. From the image attached seems pretty good! Good Work!

**Comment by luigimorel** [comment]
I need the green graph illustration but without the words. Couldn't find it in the repo.

**Reply by ZigRazor** [reply]
![Nero, Bianco e Verde Albero Southview Ricreazione Logo.png](https://user-images.githubusercontent.com/16266721/161491558-3f54262e-0a2d-49c5-a597-1ef292a58919.png)
